tests/publiccloud/suspending.pm

```diff
@@ -0,0 +1,68 @@
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: cloud-netconfig-{azure, ec2, gce}
+# Summary: This test ensures the consistency of cloud-netconfig's
+# functionality. The test shall be conducted on a VM in the cloud
+# infrastructure of a supported CSP.
+#
+# Test that a VM can be suspended and resumed without successfully
+#
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use Mojo::Base 'publiccloud::basetest';
+use Test::Assert qw(assert_equals assert_not_equals);
+use testapi;
+use utils qw(script_retry);
+use version_utils qw(is_sle);
+use publiccloud::utils qw(is_gce);
+
+sub run {
+    my ($self, $args) = @_;
+    my $instance = $args->{my_instance};
+    my $provider = $args->{my_provider};
+
+    unless (is_gce()) {
+        record_info('unsupported', 'Only GCE supports VM suspending. EC2 and Azure do hibernation but that is not supported.');
+        return 1;
+    }
+    if (is_sle('=15-SP7')) {
+        record_soft_failure('bsc#1245571 - Latest SLES 15 SP7 Image does not suspend on Google Cloud');
```

Comment:
Nitpick: A softfailure should only be raised, after it was established that the failure still occurs. A blanko softfailure is not good, because it does not allow a reviewer or anyone else to see, if the issue is still present.

***

Ignore for now, the PR is already big enough.

Reply:
The issue with SP7 is now resolved. Soft failure removed.